platforms/springboot/src/main/java/io/hawt/springboot/HawtioPlugin.java

```diff
@@ -26,10 +26,28 @@ public class HawtioPlugin {
     private Boolean bustRemoteEntryCache;
     private String pluginEntry;
 
-    public HawtioPlugin(String url, String scope, String module) {
-        this.url = url;
+    /**
+     * These are the parameters required to load a remote Hawtio plugin (a.k.a. Module Federation remote module):
```

Comment:
This is ok for the second line of the constructor javadoc comment, but the first line should first explain what does this constructor is meant to do. Explain the constructor itself rather than the parameters it accepts. The parameters are self-evident in the `@param` comments already.

See example:
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/ArrayList.html#%3Cinit%3E()